lib/matplotlib/axes/_axes.py

```diff
@@ -1975,6 +1975,8 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
         if not self._hold:
             self.cla()
         color = kwargs.pop('color', None)
+        if color is None:
```

Comment:
do you touch `bar` in that PR?

The kwarg normalization should _probably_ be done via a decorator.